DUnitX.TestFramework.pas

```diff
@@ -249,12 +259,20 @@   TTestFixtureInfoList = class(TDUnitXList<ITestFixtureInfo>, ITestFixtureInfoLi
     function GetExceptionLocationInfo : string;
     function GetExceptionAddressInfo : string;
     function GetExceptionAddress : Pointer;
+    function GetIsComparable: boolean;
+    function GetExpected: string;
+    function GetActual: string;
+    function GetFormat: TDUnitXComparableFormatClass;
 
     property ExceptionClass : ExceptClass read GetExceptionClass;
     property ExceptionMessage : string read GetExceptionMessage;
     property ExceptionLocationInfo : string read GetExceptionLocationInfo;
     property ExceptionAddressInfo : string read GetExceptionAddressInfo;
     property ExceptionAddress : Pointer read GetExceptionAddress;
+    property IsComparable: boolean read GetIsComparable;
+    property Expected: string read GetExpected;
+    property Actual: string read GetActual;
+    property Format: TDUnitXComparableFormatClass read GetFormat;
```

Comment:
> What I could do is have an ExtraInfo interface that could be queried - that would also be beneficial when and if I implement binary comparison and/or other "details" about the assertion failure that may be datatype specific. I'll explore that a little later today.

Yes, that would be way better, the current implementation is the first step to interface clumping where numerous properties are only valid given on some other property (which is imo a bad design)